scripts/module/stadium.nut

```diff
@@ -0,0 +1,44 @@
+// Stadium billboards
+
+currentModelOffset <- 0;
+BillboardsBegin <- 2042;
+billboard <- null;
+
+function SetStadiumBillboard(offset) {
+  if ( billboard ) {
+    billboard.Delete();
+  }
+  if (offset >= 0 && offset <= 2) {
+    billboard <- CreateObject(BillboardsBegin + offset, 1, -1037.08, 1340.258, 36.552, 255);
+  }
+}
+
+function onScriptLoad() {
+  SetStadiumBillboard(0);
+}
+
+function onPlayerCommand( player, cmd, text ) {
+  cmd = cmd.tolower();
+  if (! player.Admin ) {
+    return;
+  }
+  if (cmd == "ssb" || cmd == "setstadiumbillboard") {
+    if ( text ) {
+      local arg = 5;
+      if (IsNum(text)){
+        arg = text.tointeger();
+      }
+      if (arg < 3 && arg >= 0) {
+        SetStadiumBillboard(arg);
+        MessagePlayer("Changed to stadium billboard offset " + arg, player);
+      } else if (arg == -1) {
+        SetStadiumBillboard(arg);
+        MessagePlayer("The stadium billboards have been removed", player);
+      } else {
+        MessagePlayer("[#FF0000]Error: Billboard index out of range ([-1,3])", player);
```

Comment:
Range is [-1, 2].

Reply:
Thanks. It felt wrong yesterday but I could not put my finger on why.